sdk/deviceupdate/azure-iot-deviceupdate/config.yaml

```diff
@@ -0,0 +1,30 @@
+use: '@autorest/java@4.1.17'
```

Comment:
This file should be in `sdk/deviceupdate/azure-iot-deviceupdate/swagger` directory. The file should be named README.md and the contents of this file should be wrapped in \`\`\`yaml quotes. See [example here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-ingestion/swagger/README.md)